program/freedom-metal-program.wake

```diff
@@ -204,15 +164,12 @@ global def freedomMetalDUTProgramCompiler =
       | distinctBy (scmp _.getPathName _.getPathName)
 
     def outputDir = programCompileOptions.getProgramCompileOptionsOutputDir
+    def type = match dut.getDUTFPGABoardOpt
+      None     = "rtl"
+      Some opt = opt
```

Comment:
This can be replaced with the more idiomatic [`getOrElse`](https://github.com/sifive/wake/blob/2384a4e456ab7f8104e0030830f7c71f517223de/share/wake/lib/core/option.wake#L30):

```wake
def type = dut.getDUTFPGABoardOpt | getOrElse "rtl"
```